packages/tools/client-logger/app-insights-logger/src/test/ClientUtilities.ts

```diff
@@ -0,0 +1,158 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ConnectionState } from "@fluidframework/container-loader";
+import { ContainerSchema, IFluidContainer } from "@fluidframework/fluid-static";
+import {
+	TinyliciousClient,
+	TinyliciousContainerServices,
+} from "@fluidframework/tinylicious-client";
+import { FluidAppInsightsLogger } from "../FluidAppInsightsLogger";
+
+/**
+ * This module contains Fluid Client utilities, including Container creation / loading.
+ */
+
+/**
+ * Type returned from when creating / loading the Container.
+ */
+export interface ContainerLoadResult {
+	container: IFluidContainer;
+	services: TinyliciousContainerServices;
+}
+
+/**
+ * Basic information about the container, as well as the associated audience.
+ */
+export interface ContainerInfo {
+	/**
+	 * {@link ContainerInfo.container}'s unique ID.
+	 */
+	containerId: string;
+
+	/**
+	 * The initialized Fluid Container.
+	 */
+	container: IFluidContainer;
+
+	/**
+	 * Optional nickname for the Container to be used in the debugger and associated UI.
+	 */
+	containerNickname?: string;
+}
+
+function initializeTinyliciousClient(): TinyliciousClient {
+	console.log(`Initializing Tinylicious client on port ${process.env.PORT}...`);
+	return new TinyliciousClient({
+		logger: new FluidAppInsightsLogger({
```

Comment:
needing this whole application for this one line seem pretty bonkers to me. i know how it works, and it took me a while to find it.